packages/libs/bootstrap-lib/docs/launch-post.md

```diff
@@ -0,0 +1,332 @@
+Title: Release @kadena/client
+
+We've created a library that allows Javascript/Typescript users to easily
+interact with the Kadena Blockchain. We've explicitly kept out creating
+contracts as it's way more complicated to transpile Javascript to Pact, as
+contracts need to be Pact lang in the end anyway.
+
+Interaction with the Kadena Blockchain works in multiple ways, and they're
+described in this article. We'll also go into the concepts and rationale of
+@kadena/client.
+
+- [Prerequisites](#prerequisites)
+- [Bootstrapping @kadena/client](#bootstrapping-kadenaclient)
+  - [Load contracts from the blockchain](#load-contracts-from-the-blockchain)
+  - [Manually singing the transaction](#manually-singing-the-transaction)
+  - [Automated sign request to **Chainweaver desktop**](#automated-sign-request-to-chainweaver-desktop)
+- [Building a simple transaction using a template](#building-a-simple-transaction-using-a-template)
+  - [Load the contract repository](#load-the-contract-repository)
+  - [Generate code from templates](#generate-code-from-templates)
+  - [A function is generated from a template](#a-function-is-generated-from-a-template)
+  - [Using the generated code](#using-the-generated-code)
+- [Further development](#further-development)
+- [Contact the team](#contact-the-team)
+
+# Prerequisites
+
+We're not going over installation of nodejs and package management, this
+information is widely available on the internet. We will provide the bare
+minimum of the things you need to do in order to start using @kadena/client:
+
+- install nodejs version 14.x or 16.x
+- create a directory, bootstrap a `package.json` by running `npm init` or
+  `npm init -y` to use defaults
+- install the client `npm install @kadena/client`
+- install the commandline tool `npm install @kadena/pactjs-cli`
+
+# Bootstrapping @kadena/client
+
+We wanted @kadena/client to be self sufficient even when you're not using
+default contracts like `coin` or `marmalade`. That's why you have to generate
+the interfaces that are used by `@kadena/client` from information on the
+blockchain or from your own smart contracts.
```

Comment:
If I understand correctly, this is necessary for coin and marmelade as well (if not, it should be stated explicitly - "Skip this if you aren't using non-standard smart contracts)

Should we include those commonly used contracts' interfaces and include them somehow? It would remove some friction - having a simple hello-world or pay-txn up and running with the least steps possible would be better

Counterpoint: "What about when coin/marmelade is updated?"

-    Updates are always backwards compatible with respect to the interface IIRC, so existing users would not see a regression
-    We would update the upstream repo and new users could get the full functionality
-    We should include some script to refresh those definitions from chain

Reply:
> I'm thinking about leaving out marmalade alltogether as it's still in development and doesn't really add value mentioning it and only adds confusion

+1

> I want to have a kadena-templates repo that contains basic stuff for doing transactions with coin contract. Similar to the one from kda-tool.

Great!